pgcrypto/mixins.py

```diff
@@ -66,7 +66,11 @@ def get_placeholder(self, value=None, compiler=None, connection=None):
         """
         if value is None or value.startswith('\\x'):
             return '%s'
-        return self.encrypt_sql
+
+        if self.encrypt_sql.startswith("hmac"):
```

Comment:
It does not seem ideal looking for `hmac` in `self.encrypt_sql`, but the alternative would probably be another accessor (e.g. `get_encrypt_sql`) and mixin.

Reply:
Instead of breaking it out to another mixin, I added a generic `get_encrypt_sql` in the `HashMixin` and overrode that method in the `TextDigestField` implementation.